com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/v2/ApplicationCloudModelBuilder.java

```diff
@@ -58,19 +58,20 @@
     protected final DeploymentDescriptor deploymentDescriptor;
     protected final boolean prettyPrinting;
     protected final ApplicationEnvironmentCloudModelBuilder applicationEnvCloudModelBuilder;
-    protected final DeployedMta deployedMta;
+    protected final DeployedLiveModuleCalculator deployedLiveModuleCalculator;
     protected final UserMessageLogger stepLogger;
 
     protected final ParametersChainBuilder parametersChainBuilder;
 
-    public ApplicationCloudModelBuilder(DeploymentDescriptor deploymentDescriptor, boolean prettyPrinting, DeployedMta deployedMta,
-                                        String deployId, UserMessageLogger stepLogger) {
+    public ApplicationCloudModelBuilder(DeploymentDescriptor deploymentDescriptor, boolean prettyPrinting,

```

Comment:
Add a builder.
This many arguments is a bit much

Reply:
I tried to use Immutables for builder but it will be need more time to refactor due to inheritence from V3 application cloud model builder.

Reply:
Then write a custom builder